project/WeaverPlugin.scala

```diff
@@ -90,7 +90,6 @@ object WeaverPlugin extends AutoPlugin {
     "-Xlint:inaccessible",           // Warn about inaccessible types in method signatures.
     "-Xlint:infer-any",              // Warn when a type argument is inferred to be `Any`.
     "-Xlint:missing-interpolator",   // A string literal appears to be missing an interpolator id.
-    "-Xlint:nullary-override",       // Warn when non-nullary `def f()' overrides nullary `def f'.
```

Comment:
I see this was removed in 2.13.3, was it also removed in 2.12.12 ? If not, could you move it to the 2.12_only list of compiler options rather than removing altogether ?

Reply:
Moved in force push.